docs/blocks/index.md

```diff
@@ -8,10 +8,10 @@ One Block to Rule Them All
 
 Before we get started, it is important to understand that there is only ever one of each block in the game. A world consists of thousands of references to that one block in different locations. In other words, the same block is just displayed a lot of times.
 
-Due to this, a block should only ever be instantiated once, and that is during registration. Once the block is registered, you can then use the registered reference as needed. Consider this example (see the [Registration][registration] page if you do not know what you are looking at):
+Due to this, a block should only ever be instantiated once, and that is during [registration]. Once the block is registered, you can then use the registered reference as needed. Consider this example:
 
 ```java
-//BLOCKS is a DeferredRegister.Blocks
```

Comment:
This is not correct, Blocks is a specialized DR subclass, please revert this change.